deploy/lib/server_config.rb

```diff
@@ -672,16 +679,12 @@ def bootstrap
       assigns = ''
       internals = @properties['ml.system-dbs'].split ','
       internals.each do |db|
-        repnames = ''
-        repassigns = ''
-        (1..nr).each do |i|
-          repnames = repnames + %Q{
-                <replica-name>#{db}-rep#{i}</replica-name>}
-          repassigns = repassigns + %Q{
```

Comment:
The old way the code worked was to generate the replicas in the server_config.rb.  I moved the code to generate the replicas into the setup.xqy.  The config now just sends the count for the number of replicas across.